### PR TITLE
AppTP bandwidth coroutine

### PR DESCRIPTION
**Pull request by joshliebe**
Task/Issue URL: https://app.asana.com/0/0/1202254048449733/f

### Description
Starts an AppTP bandwidth coroutine (5m / 1hr) when AppTP is enabled.

### Steps to test this PR

_When AppTP is enabled_
- [x] Verify that the coroutine is triggered.

_When AppTP is disabled_
- [x] Verify that the coroutine is stopped.

### CODE REVIEW

File: bandwidth/bandwidth-impl/src/main/java/com/duckduckgo/bandwidth/impl/BandwidthPixelName.kt [hunk 1]
```diff
@@ -19,11 +19,21 @@ package com.duckduckgo.bandwidth.impl
 import com.duckduckgo.app.statistics.pixels.Pixel
 
 enum class BandwidthPixelName(override val pixelName: String) : Pixel.PixelName {
-    BANDWIDTH("m_bandwidth")
+    BANDWIDTH("m_bandwidth"),
```
Comment (aitorvs) [inline bandwidth/bandwidth-impl/src/main/java/com/duckduckgo/bandwidth/impl/BandwidthPixelName.kt:22]: Should we also remove the `RealBandwidthCollector` as part of this PR ?

Comment (joshliebe) [reply bandwidth/bandwidth-impl/src/main/java/com/duckduckgo/bandwidth/impl/BandwidthPixelName.kt:22]: I'll remove it in a follow-up PR 👍

**Review by aitorvs — COMMENTED** [review]
lgtm!

**Comment by joshliebe** [comment]
Thanks for the review @aitorvs!

**Pull request merged**